OPHD/Constants/Numbers.h

```diff
@@ -8,7 +8,7 @@
  */
 namespace constants
 {
-	inline constexpr int PlanetAnimationSpeed{50};
+	inline constexpr int PlanetAnimationSpeed{35};
```

Comment:
I guess technically we should call this a "delay" rather than a "speed". We decrease the delay to increase the speed.